homeassistant/components/unifi/switch.py

```diff
@@ -14,7 +14,6 @@
 
```

Comment:
We should be able to drop these attributes now from the Unifi PoE switch entities?

Reply:
Absolutely. Some cleanup, possible attributes and tests left